src/systemcmds/tests/test_matrix.cpp

```diff
@@ -625,7 +625,7 @@ bool MatrixTest::vector2Tests()
 	ut_test(fabs(c(1) - 0) < 1e-5);
 
 	Matrix<float, 2, 1> d(a);
-	ut_test(fabs(d(0, 0) - 1) < 1e-5);
+	ut_test(fabs(d(0, 0) - 1.0f) < 1e-5);
```

Comment:
I cannot explain why I had to make this change, and why only here.
The compiler was not helpful:
```
/run/media/julien/Data/Documents/PX4/Firmware/src/systemcmds/tests/test_matrix.cpp: In member function ‘bool MatrixTest::vector2Tests()’:
/run/media/julien/Data/Documents/PX4/Firmware/src/systemcmds/tests/test_matrix.cpp:640:1: internal compiler error: in trunc_int_for_mode, at explow.c:55
 }
 ^
Please submit a full bug report,
with preprocessed source if appropriate.
See <https://bugs.archlinux.org/> for instructions.
```